### ISSUE TITLE
Instagram not getting the right account / mixing accounts

### ISSUE THREAD

**Issue by mvfariajr** [pavelk2/social-feed#129]
My instagram is: @fariadesign  .. and the feed is getting posts from @faria_design 

any help?

Thanks!

**Comment by parachute-dev** [comment]
Hey - Also having this problem - a totally different user is being retrieved when I specifiy @<username> - any ideas?

**Comment by mvfariajr** [comment]
No, kinda let it go for now. It would be nice if we had a solution for this. Let keep this active.

**Comment by parachute-dev** [comment]
It shouldn't be too hard to correct - on line 451 onwards and use an Access
Token instead of the Client ID to get some data back. I guess the Client ID
is being used as it's easier to retrieve.

A quick dirty fix is to use the url the feed is using to see what's being
returned

https://api.instagram.com/v1/users/search/?q=<search
string>&client_id=<client id>&count=1&callback=?

If you replace the client ID with your own, and then try to search for
username, you can see what you get back.

I ended up not searching via my username, but by my profile name and
Instagram returned the right data for me. You just need to use @<search
string> for it to work with the initial parameters.

[image: Parachute] http://www.thisisparachute.com/_David R. Bova *-
*Technical
Director at Parachute_

Tel: 0141 357 4556 | Mobile: 07948 329554
david@thisisparachute.com | http://www.thisisparachute.com
http://thisisparachute.com/

[image: Twitter] http://twitter.com/thisisparachute[image: Facebook]
http://www.facebook.com/thisisparachute[image: Pinterest]
http://pinterest.com/thisisparachute
*Confidentiality Note: *This email may contain confidential and/or private
information. If you received this email in error please delete and notify
sender.
